### DISCUSSION TITLE
Get coordinates from mouse click

### DISCUSSION THREAD

**Discussion by aakshu** [alandefreitas/matplotplusplus#328] [Q&A]
I want to take a mouse click from the plot and make calculations based on the coordinates of the click, how can I do this?

**Comment by lillemakken** [comment]
I would also like to know if there are any plans to implement this in the near future or how one would be able to do so

**Comment by alandefreitas** [comment]
That would also require a new backend. See https://github.com/alandefreitas/matplotplusplus/discussions/445
